assets/large_language_models/rag/components/image_embed_index/spec.yaml

```diff
@@ -0,0 +1,43 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+tags:
+    Preview: ""
+
+version: 0.0.8
+name: llm_rag_image_embed_index
+display_name: LLM - Embedding images with Florence 
+is_deterministic: true
+
+description: "Embeds input images and stores it in Azure Cognitive Search index with metadata using Florence embedding resource. MLIndex is stored to `output_path`."
+outputs:
+  output_path:
+    type: uri_folder
+    description: Where to store the output MLIndex.
+inputs:
+  input_path:
+    type: string
+    optional: False
+    description: input dataset of images and json metadata.
+  florence_connection_id:
```

Comment:
This string is visible to customer. I'm not sure if we use "florence" appropriate, maybe we can give it a more generic name? Is this an unique id? Can we just use embedding_connection_id?

Reply:
Yes, I fixed it.